odoo/models.py

```diff
@@ -3823,8 +3823,12 @@ def create(self, vals_list):
                 for parent, data in zip(parents, parent_data_list):
                     data['stored'][parent_name] = parent.id
 
-        # create records with stored fields
-        records = self._create(data_list)
+        # create records with stored fields, dropping `default_{field}` keys
+        # from the env for sub-field creations as well as return
+        records = self.with_context({
+            k: v for k, v in self.env.context.items()
+            if not k.startswith('default_')
+        })._create(data_list)
```

Comment:
You can use: https://github.com/odoo/odoo/blob/fa576fc74e904ea428fd9cd24070ddbb64c553b4/odoo/tools/misc.py#L980-L982

Reply:
Nice suggestion, thanks, and I can even easily update it to take an exclusion predicate for filtering out `active_test`.